safeeyes/ui/break_screen.py

```diff
@@ -55,6 +55,10 @@ def __init__(self, context, on_skipped, on_postponed, style_sheet_path):
         self.shortcut_disable_time = 2
         self.strict_break = False
         self.windows = []
+        self.is_wayland = os.environ.get('XDG_SESSION_TYPE') == 'wayland'
```

Comment:
This should not access the environment again, the information can already be found in `self.context`.